<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserMainWindow</class>
 <widget class="QMainWindow" name="UserMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="usernoLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AlArabiya</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>label</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>261</width>
      <height>391</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>contact</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>264</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionModifyUserinfo"/>
    <addaction name="actionAddFriend"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionModifyUserinfo">
   <property name="text">
    <string>Modify User Info</string>
   </property>
  </action>
  <action name="actionAddFriend">
   <property name="text">
    <string>Add Friend</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
